lib/ansible/modules/database/mysql/mysql_info.py

```diff
@@ -25,8 +25,8 @@
   filter:
     description:
     - Limit the collected information by comma separated string or YAML list.
-    - Allowable values are C(version), C(databases), C(settings), C(users),
-      C(slave_status), C(slave_hosts), C(master_status), C(engines).
+    - Allowable values are C(version), C(databases), C(settings), C(status),
```

Comment:
```suggestion
    - Allowable values are C(version), C(databases), C(settings), C(global_status),
```

Reply:
Agree, I have renamed it to `global_status`

Reply:
good job! thank you